automated install/basic-install.sh

```diff
@@ -1940,14 +2110,14 @@ Your Admin Webpage login password is ${pwstring}"
     fi
 
     # Final completion message to user
-    whiptail --msgbox --backtitle "Make it so." --title "Installation Complete!" "Configure your devices to use the Pi-hole as their DNS server using:
-
-IPv4:	${IPV4_ADDRESS%/*}
-IPv6:	${IPV6_ADDRESS:-"Not Configured"}
-
-If you have not done so already, the above IP should be set to static.
-
-${additional}" "${r}" "${c}"
+    dialog --no-shadow --clear \
```

Comment:
Why is the background purple here?

Reply:
I'm not seeing a purple background?

Reply:
Do you have a file named `.dialogrc` in your home dir?

Reply:
Shouldn't the backtitle be "_Installation Complete!_"?

Reply:
No `dialogrc`

Reply:
Reason for the purple background was that the installer used the old `whiptail` from the `master` branch at this point.
If one runs a locally modified installer it will start with the new code, will clone the repo to `/etc/.pihole` and then reach `finalExports()` where it sources `webpage.sh` from the local repo. One of the first things it does, is to source `basic-install.sh` from the local repo - which is at this point the current `master`. The script will therefore use the old code from this point onwards.